sysadmin/templates/nginx_conf.j2

```diff
@@ -0,0 +1,374 @@
+$ANSIBLE_VAULT;1.2;AES256;oc
```

Comment:
TO DO: extract out the sensitive stuff (like how we block certain requests) from the parts that would be very helpful  to be plain text

Reply:
ok